cli/azd/docs/environment-variables.md

```diff
@@ -0,0 +1,19 @@
+# Environment Variables
+
+Environment variables that can be used to configure `azd` behavior. For environment variables that accept a boolean, the values `1, t, T, TRUE, true, True` are accepted as "true"; the values: `0, f, F, FALSE, false, False` are all accepted as "false".
```

Comment:
Should we mention if users can set these variables per azd-environment or not?

Reply:
Created a section to clarify -- these are environment variables used with azd -- not azd environments. We may track that as a separate section within the same doc in the future.